### ISSUE TITLE
Remove mp3 files from GitHub

### ISSUE THREAD

**Issue by BasThomas** [Contravariance/website#16]
It starts feeling slow and clunky.

**Comment by terhechte** [comment]
I can create a subdomain on contravariance.rocks, however the question is mostly deployment. Github makes deployment for both of us easy. I suppose I could add a restricted ssh / sftp account for contravariance.rocks or something like that

**Comment by BasThomas** [comment]
Yeah true. Thought the same. We can leave it as-is for the time being but it would be nice to find a solution for the future :) 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>
